cli/lsp/config.rs

```diff
@@ -1700,6 +1700,7 @@ impl ConfigTree {
     if specifier.path().ends_with("/deno.json")
       || specifier.path().ends_with("/deno.jsonc")
       || specifier.path().ends_with("/package.json")
+      || specifier.path().ends_with("/.package-lock.json")
```

Comment:
The lockfile name is `package-lock.json` without the leading dot.

```suggestion
      || specifier.path().ends_with("/package-lock.json")
```

Should we watch the other package manager's lockfiles too like `yarn.lock`, `pnpm-lock.yaml` or `bun.lockb`?

Reply:
There is this `node_modules/.package-lock.json` that this is supposed to watch, since it tracks population of `node_modules`. Both might work? But this one seems more apt.

Reply:
@nayeemrmn Maybe expliclty check for `node_modules/.package-lock.json`? Also `node_modules/.yarn-integrity`, `.modules.yaml` (pnpm), and `node_modules/.deno/.setup-cache.bin`.